chart/templates/NOTES.txt

```diff
@@ -1 +1,5 @@
 1. The kapel accounting cron job should be running now. When the kapel pod runs check the logs of the processor and ssmsend containers.
+
+{{ if not .Values.processor.config.PROCESSORS -}}
+WARNING: processor.config.PROCESSORS is empty in values.yaml. Ensure that workload pods have a cpu resource request set.
```

Comment:
I understand wanting to highlight this, but this is also working as designed and documented so should not really be a warning.
What about something like this?
"NOTE: processor.config.PROCESSORS is not set by default. Ensure that workload pods have a CPU resource request set or they will not be accounted."

Reply:
Yeah, that update in terminology makes sense. I can also add a guard so this only prints if in gratia output mode since it isn't a concern in apel mode.

Reply:
Just realized that this NOTES.txt also depends on processors.config being changed from a string to an object via https://github.com/rptaylor/kapel/pull/65 . We should hold off on merging this change until that's complete